wine.py

```diff
@@ -30,24 +27,13 @@ def get_pids_using_file(file_path, mode=None):
             pass
     return pids
 
-def wait_process_using_dir(directory):
-    logging.info(f"* Starting wait_process_using_dir for {directory}…")
-
-    # Get pids and wait for them to finish.
-    pids = get_pids_using_file(directory)
-    for pid in pids:    
-        logging.info(f"wait_process_using_dir PID: {pid}")
-        psutil.wait(pid)
-
-    logging.info("* End of wait_process_using_dir.")
-
```

Comment:
Is there a reason this is removed?

Just unused?

Reply:
Ha, for some reason the same function was both imported from `utils.py` to and then defined in `wine.py`. So this definition was redundant.